src/Forms/Ocr/VobSubOcr.cs

```diff
@@ -637,7 +637,7 @@ internal void InitializeBatch(string vobSubFileName, VobSubOcrSettings vobSubOcr
                 text = text.Replace(Environment.NewLine + " ", Environment.NewLine);
 
                 // max allow 2 lines
-                if (checkBoxAutoBreakLines.Checked && text.Replace(Environment.NewLine, "*").Length + 2 <= text.Length)
+                if (checkBoxAutoBreakLines.Checked && Utilities.CountTagInText(text, Environment.NewLine) <= 2)
```

Comment:
Perhaps `GetNumberOfLines` is more "correct" ?

Reply:
Done.